src/main/resources/site/content-types/content-data-locale-fallback/content-data-locale-fallback.xml

```diff
@@ -0,0 +1,80 @@
+<content-type xmlns="urn:enonic:xp:model:1.0">
+    <display-name>Oversettelser for ikke-lokalisert innhold</display-name>
+    <description>Benyttes ved behov for oversetting av enkelt-felter på innhold som ikke er lokalisert, f.eks. for
+        utlisting i oversikter
+    </description>
+    <super-type>base:structured</super-type>
+    <is-abstract>false</is-abstract>
+    <is-final>true</is-final>
+    <allow-child-content>true</allow-child-content>
+    <form>
+        <item-set name="items">
+            <label>Oversettelser (listen autogenererer inntil 500 elementer)</label>
```

Comment:
Hva er årsaken til grensen på 500? (Mest fordi jeg er nysgjerrig)

Reply:
Content Studio blir veldig tregt (og krasjer etter hvert) med for mange liste-elementer. Så jeg la inn en begrensing som jeg tror skal være nok for alle realistiske behov (ikke at det er noe spesielt med tallet 500, det var litt tilfeldig valgt).